juntagrico/entity/jobs.py

```diff
@@ -168,7 +168,7 @@ class Job(JuntagricoBasePoly):
 
     @property
     def type(self):
-        raise NotImplementedError
+        raise NotImplementedError()
```

Comment:
I think this is not necessary. And there might be other places where this is done without braces, so it should be kept consistent in this project.

Reply:
Ah, yes, I didn't know you that if you say `raise SomeClass` it gets implicitely converted to `raise SomeClass()`.